libraries/dynamic-ui-react/src/style/components/_d-monthpicker.scss

```
@@ -41,6 +42,11 @@
     font-weight: var(--#{$prefix}monthpicker-font-weight);
     color: var(--#{$prefix}monthpicker-header-color);
     background-color: var(--#{$prefix}monthpicker-header-bg);
+    .btn-sm {
+      --#{$prefix}btn-padding-x: 0px;
+      --#{$prefix}btn-padding-y: 0px;
```

```suggestion
      --#{$prefix}btn-padding-x: 0;
      --#{$prefix}btn-padding-y: 0;
```

cero no tiene unidad :)